### ISSUE TITLE
The tooltip.renderer of chartThemeOverrides not work in cross-filter chart

### ISSUE THREAD

**Issue by YuerLee** [ceolter/ag-grid#5576]
**I'm submitting a ...** 
```
[x] bug report => see 'Providing a Reproducible Scenario'
[ ] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[ ] support request => see 'Requesting Community Support'
```

**Current behavior**
Cannot use `tooltip.renderer` in `chartThemeOverrides`in the cross-filter chart.

[plnkr](https://plnkr.co/edit/M09tAmRKh00QgeLI?preview)

![image](https://user-images.githubusercontent.com/2075566/191697420-b8afc776-032d-4990-91f5-cea231a70423.png)

**Expected behavior**
It works if version < v28.

[plnkr with v27.3.0](
https://plnkr.co/edit/2hjQDvUvCemA51ao?preview)

![image](https://user-images.githubusercontent.com/2075566/191698035-a78697ae-a8ba-43c2-a3e6-068bf0c09a5f.png)

And also works in the range chart

[plnkr](https://plnkr.co/edit/gflrm2y9KAFmFa0m?open=main.js&preview)

![image](https://user-images.githubusercontent.com/2075566/191701617-f73f525c-8f5f-4105-8ab6-db6624687402.png)

* **AG Grid version:** 28.1.3

* **Browser:**
```
'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36'
```
* **Language:** ES6

**Comment by alantreadway** [comment]
Hi @YuerLee,

Thank you for reporting this - much appreciated! Based on your description we've now reproduced this issue as a bug.

We have added this bug to our backlog and we are tracking it with the following reference and description:
**AG-7550 - [Charts] Cross-filtering Pie chart `tooltip.renderer` not being applied.**

We try to fix bugs from one release to the next, so this should be fixed in the next release or the one after if it was raised too close to the next release date.

See whether this item will be in the next release by checking the NEXT RELEASE checkbox on the product pipeline page:
https://www.ag-grid.com/ag-grid-pipeline/

The best way to track this is to sign up for AG Grid new release notifications using [the instructions here](https://ag-grid.zendesk.com/hc/en-us/articles/360002648058-How-do-I-sign-up-for-ag-Grid-release-notifications-). This way you'll know as soon as a new version is out and you can check whether this specific item was implemented on [the changelog page](https://www.ag-grid.com/ag-grid-changelog/).

Thanks again for bringing this up with us.

Regards,
Alan Treadway

**Comment by YuerLee** [comment]
Fixed in v29.0.0.
